R/auditCommonFunctionsBayesian.R

```diff
@@ -19,1941 +19,906 @@
+			mPrior 	<- data.frame(x = popK, 		y = jfa:::.dBetaBinom(popK, N = N - n + k, shape1 = alphaParameterPrior, shape2 = betaParameterPrior))
+			ePrior 	<- data.frame(x = epopK, 		y = jfa:::.dBetaBinom(epopK, N = N - n + k, shape1 = alphaParameterPrior, shape2 = betaParameterPrior))
+			
+			# Calculate upper bounds for beta-binomial prior and posterior
+			bPrior 	<- jfa:::.qBetaBinom(confidence, N = N - n + k, shape1 = alphaParameterPrior, shape2 = betaParameterPrior) 
+			bPost 	<- jfa:::.qBetaBinom(confidence, N = N - n + k, shape1 = alphaParameterPosterior, shape2 = betaParameterPosterior)
+		}
+
+		extra <- base::switch(options[["shadePrior"]],
+								"shadePriorCredibleRegion" = data.frame(x = 0, y = 0, l = "1"),
+								"shadePriorHypotheses" = data.frame(x = c(0, 0), y = c(0, 0), l = c("1", "2")))
+		shade <- base::switch(options[["shadePrior"]],
+								"shadePriorCredibleRegion" = rgb(0.5, 0.8, 0.57, .7), # Lightgreen
+								"shadePriorHypotheses" = c(rgb(0, 0.64, 0.80, .6), rgb(0.98, 0.34, 0, .6))) # Blue and orange
+
+		dPlot <- dPrior
+		guide <- FALSE
+		scaleValues <- "dashed"
+
+		if(options[["priorPlotExpectedPosterior"]]){  
+			dPlot <- rbind(dPlot, dPost)
+			dPlot$type <- factor(dPlot$type, levels = levels(dPlot$type)[c(1,2)])
+			scaleValues <- c(scaleValues, "dotted")
+			guide <- ggplot2::guide_legend(nrow = 1, byrow = FALSE, title = "", order = 1)
+		}
+
+		xBreaks <- jaspGraphs::getPrettyAxisBreaks(xseq, min.n = 4)
+		yBreaks <- c(0, 1.2 * max(dPlot$y))
+
+		plot <- ggplot2::ggplot(data = dPlot, mapping = ggplot2::aes(x = x, y = y)) +
+				ggplot2::geom_line(mapping = ggplot2::aes(x = x, y = y, linetype = type), lwd = 1) +
+				ggplot2::scale_linetype_manual(values = scaleValues, guide = guide) +
+				ggplot2::scale_y_continuous(name = gettext("Density"), breaks = yBreaks, labels = c("", ""), limits = range(yBreaks))
+
+		if(likelihood == "hypergeometric"){
+			plot <- plot + ggplot2::scale_x_continuous(name = gettextf("Population misstatement (%1$s)", "\u03B8"), breaks = xBreaks, limits = range(xBreaks), labels = xBreaks)
+		} else {
+			title <- ifelse(options[["separateKnownAndUnknownMisstatement"]], yes = gettextf("Unseen population misstatement (%1$s)", "\u03B8"), no = gettextf("Population misstatement (%1$s)", "\u03B8"))
+			plot <- plot + ggplot2::scale_x_continuous(name = title, breaks = xBreaks, limits = range(xBreaks), labels = paste0(xBreaks * 100, "%"))
+		}
+		
+		if(options[["priorPlotAdditionalInfo"]]){
+			if(options[["shadePrior"]] != "shadePriorNone"){ 
+				plot <- plot + ggplot2::geom_point(data = extra, mapping = ggplot2::aes(x = x, y = y, shape = l), size = 0, color = shade)
+				
+				if(options[["shadePrior"]] == "shadePriorCredibleRegion"){
+					if(options[["priorPlotExpectedPosterior"]]){
+						plot <- plot + ggplot2::scale_shape_manual(name = "", values = 21, labels = gettextf("%1$s%% Credible \nregion for %2$s", round(confidence * 100, 2), "\u03B8"))
```

Comment:
Why the newline in the label (`\n`)? The next else statement does not have it

Reply:
Ah yes, otherwise the legend gets too full